S23/sandeep-1-kumar/LowerCasing.java

```diff
@@ -0,0 +1,60 @@
+import java.util.Locale;
+
+public class LowerCasing {
+
+    public static void main(String[] args)
+    {
+
+    }
+    public String lowerCaseConversion(String languageType,String word)
+    {
+        String[] exceptedLanguages = {"zh" , "ja" , "th"};
+        Locale locale = Locale.forLanguageTag(languageType);
+        String language = locale.getLanguage();  
+        for(String element : exceptedLanguages)
+        {
+            if(element.equals(language))
+            {
+                return word;
+            }
+        }
+        String returningValue = word;
+        if(language.equals("tr") || language.equals("az"))
+        {
+            returningValue = word.replace('I' ,'\u0131');
+        }
+        else if(language.equals("el"))
+        {
+            if (word.endsWith("Σ"))
+            {
+                returningValue = word.substring(0, word.length() - 1) + "\u03c2"; 
+            }
+            else
+            {
+                returningValue = word.replace("Σ", "σ");
+            }
+        }
+        else if(language.equals("ga"))
+        {
+            if((word.charAt(0) == 'n' || word.charAt(0) == 't'))
+            {
+                char[] vowelArray = {'A','E','I','O'};
```

Comment:
can you confirm this as all test cases are not being handled uppercase Irish vowel (A,E,I,O,U,Á,É,Í,Ó,Ú),

Reply:
Yes , Thanks I have added the Irish vowel cases.

Reply:
great!